server/routes/script.js

```diff
@@ -1,9 +1,59 @@
 const express = require('express');
 const router = express.Router();
+const axios = require('axios');
+
+const url = process.env.api;
 
 router.get('/', (req, res) => {
-    res.render('home');
+    function showMovies(url, page = 1) {
+
+        axios.get(url + page)
+            .then(response => {
+                res.render('home', { appl: response.data.results });
+            }, error => {
+                console.log(error);
+            });
+
+    }
+    showMovies(url);
+
 })
 
+router.post('/search', (req, res) => {
+    const { search } = req.body;
+    const SEARCHAPI = process.env.searchapi;
+
+    if (search) {
+
+        axios.get(SEARCHAPI + search)
```

Comment:
After parsing environment variables into an object, as I mentioned in a comment above, you can do something like: 
`axios.get(${config.themoviedb.host}/search/movie?&api_key=04c35731a5ee918f014970082a0088b1&query=, .......`